src/main/java/nextstep/ladder/utils/StringUtils.java

```diff
@@ -0,0 +1,12 @@
+package nextstep.ladder.utils;
+
+public class StringUtils {
+    private static final String DELIMITER = ",";
+
+    private StringUtils() {
+    }
```

Comment:
private으로 생성자를 통한 인스턴스화를 막아주셨네요 👍
내부에서도 생성하지 못하도록 예외를 던져보면 어떨까요?
이펙티브 자바의 아이템 4번을 참고해보시면 좋을 것 같아요!
```suggestion
    private StringUtils() {
        throw new AssertionError();
    }
```

Reply:
이런 방법은 모르고있었네요 ㅎㅎ 이펙티브 자바읽어보도록 해야할것 같아요